pkg/input/input.go

```diff
@@ -39,3 +40,31 @@ func Password(promptText string) (string, error) {
 	fmt.Print("\n")
 	return string(bytePassword), nil
 }
+
+// Structure to hold content required for displayig prompts required for promptui library functions
```

Comment:
nit: typo change displayig to displaying

Reply:
```suggestion
// Structure to hold content required for displaying prompts required for promptui library functions
```